src/nxdoc/nuxeo-server/installation/tomcat-server-zip/installing-the-nuxeo-platform-on-linux.md

```diff
@@ -1,364 +1,11 @@
-        message: Migration of unmigrated content due to installation of a new plugin
-        version: '15'
-    -
-        author: Solen Guitter
-        date: '2013-01-10 12:12'
-        message: ''
-        version: '14'
-    -
-        author: Solen Guitter
-        date: '2012-09-05 00:55'
-        message: Migration of unmigrated content due to installation of a new plugin
-        version: '13'
-    -
-        author: Solen Guitter
-        date: '2012-09-05 00:55'
-        message: Migration of unmigrated content due to installation of a new plugin
-        version: '12'
-    -
-        author: Solen Guitter
-        date: '2012-09-05 00:55'
-        message: Migrated to Confluence 4.0
-        version: '11'
-    -
-        author: Solen Guitter
-        date: '2012-09-05 00:55'
-        message: ''
-        version: '10'
-    -
-        author: Mathieu Guillaume
-        date: '2012-07-31 21:42'
-        message: Updated ubuntu distribution name to latest LTS
-        version: '9'
-    -
-        author: Thierry Martins
-        date: '2012-04-26 10:04'
-        message: ''
-        version: '8'
-    -
-        author: Solen Guitter
-        date: '2012-03-28 17:32'
-        message: ''
-        version: '7'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 11:47'
-        message: ''
-        version: '6'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 00:29'
-        message: ''
-        version: '5'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 00:03'
-        message: ''
-        version: '4'
-    -
-        author: Solen Guitter
-        date: '2011-12-13 00:01'
-        message: Added toc
-        version: '3'
-    -
-        author: Solen Guitter
-        date: '2011-12-07 11:31'
-        message: ''
-        version: '2'
-    -
-        author: Solen Guitter
-        date: '2011-12-07 11:21'
-        message: ''
-        version: '1'
-
+  date: '2021-01-29'
+  status: ok
+toc: false
 ---
 
-## Checking Requirements
-
-First, have a look at the detailed steps to [check your Java version]({{page page='tomcat-server-zip'}}#java-check) and install it.
-
-## Installing the Nuxeo Platform From the ZIP Archive
-
-### Install the Nuxeo Platform
-
-Download the ZIP archive from the [Nuxeo distribution download site](https://connect.nuxeo.com/nuxeo/site/connect/distributions) and unzip it using your favorite tool.
-
-{{#> callout type='warning' heading='PRIVATE DISTRIBUTION'}}
-You should contact your Nuxeo Administrator or Nuxeo sales representative to get access to this distribution.
-{{/callout}}
-
-### Installation Path
-
-The usual directory into which install the server is `/opt`.
-
-### What's Next?
-
-Do you want to evaluate the platform? You can now [start the server]({{page page='server-start-and-stop'}}).
-You want to do a complete installation, that is compatible for a production environment? You should [prepare your environment]({{page page='setup-best-practices'}}) now.
```

Comment:
Don't you want to keep advertising for these 2 pages "Server Start and Stop" and "Setup Best Practices" in the ZIP distribution page?